packages/web-api-client/src/a11y-service-client.spec.ts

```diff
@@ -42,35 +46,34 @@ describe(A11yServiceClient, () => {
         const response = { statusCode: 200 };
         const requestBody = [{ url: scanUrl, priority }];
         const options = { json: requestBody };
-        requestMock
-            .setup(req => req.post(`${baseUrl}/scans`, options))
-            // tslint:disable-next-line: no-any
-            .returns(() => Promise.resolve(response) as any)
+        postMock
+            .setup(req => req(`${baseUrl}/scans`, options))
+            .returns(async () => Promise.resolve(response))
             .verifiable(Times.once());
         const actualResponse = await testSubject.postScanUrl(scanUrl, priority);
         expect(actualResponse).toEqual(response);
     });
 
     it('getScanStatus', async () => {
-        const scanId = 'scanid';
+        const scanId = 'scanId';
         const response = { statusCode: 200 };
-        requestMock
-            .setup(req => req.get(`${baseUrl}/scans/${scanId}`))
-            // tslint:disable-next-line: no-any
-            .returns(() => Promise.resolve(response) as any)
+
+        getMock
```

Comment:
add 1 more test when priority is not set. (make sure all the branches are covered by uts)